packages/ui-new/src/composites/wui-list-select-account/index.ts

```diff
@@ -0,0 +1,78 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import styles from './styles.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import '../../components/wui-text/index.js'
+import '../../components/wui-icon/index.js'
+import '../wui-avatar/index.js'
+import { UiHelperUtil } from '../../utils/UiHelperUtil.js'
+import type { IconType } from '../../utils/TypeUtil.js'
+
+@customElement('wui-list-select-account')
+export class WuiListSelectAccount extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() address = ''
+
+  @property() description = 'Email'
+
+  @property() icon: IconType = 'mail'
+
+  @property({ type: Number }) dollarAmount = 0
```

Comment:
For future features, maybe we would start showing different currencies depending on the user's choice, what about separating `amount` and `currency` and we would keep the currency USD by default for now?

Reply:
Good idea. Let me do that.